support-frontend/assets/pages/supporter-plus-landing/components/summaryTsAndCs.tsx

```diff
@@ -119,9 +117,7 @@ export function SummaryTsAndCs({
 			</>
 		),
 		TierThree: summaryTsAndCsTierThreeGuardianAdLite,
-		DigitalSubscription: (
-			<>{enablePremiumDigital && summaryTsAndCsTierThreeGuardianAdLite}</>
-		),
+		DigitalSubscription: <>{summaryTsAndCsTierThreeGuardianAdLite}</>,
```

Comment:
Maybe worth renaming `summaryTsAndCsTierThreeGuardianAdLite` while in here? Maybe to something reflecting the content of the copy (rather than which products use it) - something like `autoRenewUtilCancelTsAndCs`?